File: C/Sequencial/1003.c
/**
 * @file 1003.c
 * @author michaelthierry86@gmail.com
 * @brief Soma Simples: Ese programa calcula a soma de dois numeros inteiros
 * Ele le dois valores inteiros do usuario, soma e mostra o resultado na tela 
 * @version 0.1
 * @date 2022-02-27
 * 
 * @copyright Copyright (c) 2022
 * 
 */
/*Incluindo Biblioteca E/S*/
#include <stdio.h>
/*Função Principal*/ 
void main(){
    //Declarando variaveis
    int valor_A, valor_B, soma;
    //Lendo do usuario
    scanf("%i", &valor_A);
    scanf("%i", &valor_B);
    //Somando os valores
    soma = valor_A + valor_B;
    //Mostrando o resultado
    printf("SOMA = %i\n", soma);
    
    return 0;
}

File: JavaScript/Sequencial/1003/1003.js
/**
 * Programa: 1003 - Soma Simples: 
 * Esse programa recebe dois valores
 * Em seguida ele soma os dois valores e mostra o resultado
 */
/*Lendo do arquivo*/
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');
//Pegando os valores
var valor_A = parseInt(lines.shift());
var valor_B = parseInt(lines.shift());
//Somando os valores
var soma = valor_A + valor_B;
//Mostrando o resultado
console.log('SOMA = '+soma);